db/migrate/20160729182517_remove_system_add_source_to_miq_ae_namespace.rb

```diff
@@ -0,0 +1,33 @@
+class RemoveSystemAddSourceToMiqAeNamespace < ActiveRecord::Migration[5.0]
+  class MiqAeNamespace < ActiveRecord::Base; end
+
+  def up
+    say_with_time('Migrating System attribute to Source in MiqAeNamespace') do
+      add_column :miq_ae_namespaces, :source, :string
+
+      MiqAeNamespace.where(:parent_id => nil).each do |obj|
```

Comment:
@mkanoor This should be wrapped in a say_with_time...can you make a separate PR to cleanup?